kornia/augmentation/_2d/geometric/resize.py

```diff
@@ -114,6 +114,12 @@ def __init__(
             p=p,
         )
 
+    def apply_transform(
+        self, input: Tensor, params: Dict[str, Tensor], flags: Dict[str, Any], transform: Optional[Tensor] = None
+    ) -> Tensor:
+        self._param_generator = rg.ResizeGenerator(resize_to=self.flags['size'], side=self.flags['side'])
+        return super().apply_transform(input, params, flags, transform)
+
```

Comment:
Add some check here before recompute the params?

Reply:
Try cache at least the last one, like check if the last aspectio ratio and long/small side is equals the actual

Reply:
Depends on the case, that's the reason I don't know the best case to assume here

If we always need to check and recompute, just recompute will be the best case -- but if we never needs to recompute we can save some operations -- here https://github.com/kornia/kornia/blob/598444893212899f6f3678f45a10b5315ef0f5f9/kornia/augmentation/random_generator/_2d/resize.py#L46

Reply:
Ideally. We shall not call the random generator again. There must be somewhere in the code that weirdly remembers the parameters.

Reply:
then, should we wait to find the issue to merge; or we proceed and fix later ?

Reply:
I will investigate this further in the next few days